runtime/common/src/main/kotlin/kotlinx/serialization/json/JSON.kt

```diff
@@ -17,15 +17,15 @@
 package kotlinx.serialization.json
 
 import kotlinx.serialization.*
-import kotlin.reflect.KClass
+import kotlin.reflect.*
 
 data class JSON(
-        private val unquoted: Boolean = false,
-        private val indented: Boolean = false,
-        private val indent: String = "    ",
-        internal val nonstrict: Boolean = false,
-        val updateMode: UpdateMode = UpdateMode.OVERWRITE,
-        val context: SerialContext? = null
+    private val unquoted: Boolean = false,
+    private val indented: Boolean = false,
+    private val indent: String = "    ",
+    internal val strictMode: Boolean = true,
```

Comment:
Can we afford such breaking change in API?

Reply:
There will be at least one supporting release (together with 1.2.60), so seems this should be postponed until core refactoring?

Reply:
Yes, I'm fine with it